ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroDataTranslator.java

```diff
@@ -117,7 +119,9 @@ private static Object replaceSchema(final Schema schema, final Object object) {
 
       case STRUCT:
         return convertStruct((Struct) object, schema);
-
+      case BYTES:
```

Comment:
How did we reach the assumption that given type of `BYTES` would require a decimal conversion?

Reply:
Decimals in Avro are represented as bytes. [This function](https://github.com/confluentinc/ksql/blob/a321a753a08354d10d4166366c3fe9b13517a258/ksqldb-common/src/main/java/io/confluent/ksql/util/DecimalUtil.java#L43) is called earlier on to reject any `BYTES` that don't represent decimals. Right now, we don't have a `BYTES` type in ksqlDB so all incoming bytes must be decimals, but this logic will change once we implement it.